docs/flax.linen.rst

```diff
@@ -3,7 +3,9 @@ flax.linen package
 ==================
 
 .. currentmodule:: flax.linen
-.. automodule:: flax.linen
+
+Linen is the Flax Module system. Read more about our design goals in the `Linen README <https://github.com/google/flax/blob/master/flax/linen/README.md>`_.
```

Comment:
while reading through that file: what happened to the "various advanced use-cases" link?
https://github.com/google/flax/tree/master/linen_examples/core_design_test

Reply:
It's still there? see the bottom -- ""Design tests" used to ensure that our "functional core" supports various advanced use-cases, and that the mostly-syntactic-sugar Module abstraction doesn't get in the way"